java/client/src/org/openqa/selenium/remote/tracing/Tags.java

```diff
@@ -65,4 +65,37 @@ private Tags() {
       span.setStatus(Status.UNKNOWN);
     }
   };
+
+  public static final BiConsumer<Map<String, EventAttributeValue>, HttpRequest>
+      HTTP_REQUEST_EVENT =
+      (map, req) -> {
+        map.put(AttributeKey.HTTP_METHOD.toString(),
```

Comment:
I've seen a lot of `.toString` being used in this method. Why not simply have the key of the map be an `AttributeKey`?

Reply:
Update the `AttributeKey` enum to have a getter instead. Since this stores commonly used constants for the key. The `Map<String, EventAttributeValue>` is used since keys can be retrieved from the AttributeKey enum or they can be any other strings.